### DISCUSSION TITLE
CPU requirements by performance rating?

### DISCUSSION THREAD

**Discussion by ramseyrt** [Security-Onion-Solutions/securityonion#8986] [Unsupported Versions] [answered]
Hello,

I'm new to Security Onion and am currently looking over the hardware requirements listed [here](https://docs.securityonion.net/en/2.3/hardware.html).  The document provides CPU requirements language like "8 CPU cores at a minimum".  Unfortunately, that's a bit vague considering a Xeon Bronze 3106 (Passmark Score 5,765) CPU and a Xeon Gold 6334 (Passmark Score 22,160) CPU are both 8 cores but have vastly different performance ratings.  Are there any performance related hardware requirements listed anywhere?

Thanks in advance!

Rob

**Comment by dougburks** [comment, marked as answer]
> I'm new to Security Onion

Welcome!

> The document provides CPU requirements language like "8 CPU cores at a minimum". Unfortunately, that's a bit vague considering a Xeon Bronze 3106 (Passmark Score 5,765) CPU and a Xeon Gold 6334 (Passmark Score 22,160) CPU are both 8 cores but have vastly different performance ratings. Are there any performance related hardware requirements listed anywhere?

These requirements may seem vague because they are **minimum** requirements. Actual requirements depend on many variables in your environment. `8 CPU cores at a minimum` comes from [this section](https://docs.securityonion.net/en/2.3/hardware.html#manager-node-with-local-log-storage-and-search) which is discussing a manager node with local log storage and search. For this use case, higher passmark scores are better of course, but probably more important is having plenty of RAM and fast disk (preferably SSD for best results).

(If you're looking to purchase hardware, please note that we do sell [appliances](https://securityonionsolutions.com/hardware/) that are already preconfigured for you so that you don't have to figure all of this out.)

**Reply by ramseyrt** [reply]
Hello Doug,

Thanks for the reply.  I did look at the appliance section, hoping to find some CPU data for comparison sake.  Although the appliances provide capability data related to their function, they don't provide any technical detail about the system internals.

We were previously running Security Onion on an old system with a Xeon E5440 (4 Core)/16gb RAM/2TB HD running Ubuntu 14 another system running with a Xeon E5-2620 v4 (8 Core)/32GB RAM/2TB HD running Ubuntu 18.  When I found both systems, they had been abandoned.  While I could certainly attempt a re-install of the OS and software, I thought I'd try and verify the old hardware was adequate for the current version of the software.

Thanks!

Rob
